blocks/v2-truck-lineup/v2-truck-lineup.css

```diff
@@ -0,0 +1,308 @@
+  display: flex;
+  flex-flow: row nowrap;
+  padding-left: 0;
+  margin: 0;
+}
+
+.v2-truck-lineup__description-container {
+  order: 2;
+  overflow: hidden;
+}
+
+.v2-truck-lineup__images-container {
+  align-items: flex-end;
+  overflow: scroll hidden;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+  scrollbar-width: none;
+}
+
+.v2-truck-lineup__images-container::-webkit-scrollbar {
+  display: none;
+}
+
+.v2-truck-lineup__image-item {
+  flex: none;
+  scroll-snap-align: center;
+  text-align: center;
+  width: var(--truck-lineup-img-width);
+}
+
+.v2-truck-lineup__image-item picture {
+  display: block;
+  animation-duration: 1s;
+  animation-delay: 0.5s;
+  animation-name: truck-entry;
+  animation-timing-function: ease-in;
+}
+
+.v2-truck-lineup__image-item:first-child {
+  margin-left: 50vw;
+}
+
+.v2-truck-lineup__images-container::after {
+  content: '';
+  display: block;
+  flex: 0 0 50vw;
+}
+
+.v2-truck-lineup__image-item img {
+  aspect-ratio: 16 / 9;
+  max-height: 80vh;
+  width: auto;
+}
+
+.v2-truck-lineup__content {
+  margin: 0 auto 32px;
+  text-align: center;
+  width: 90%;
+}
+
+.v2-truck-lineup__desc-item {
+  color: var(--text-color);
+  flex: none;
+  opacity: 0;
+  width: 100%;
+
+  /* fadeout */
+  transition: opacity 0.3s cubic-bezier(0, 0, 0, 1);
+}
+
+.v2-truck-lineup__desc-item.active {
+  opacity: 1;
+  transition: opacity 0.5s cubic-bezier(0, 0, 0, 1) 0.5s;
+}
+
+.v2-truck-lineup__text {
+  margin: 0 auto;
+  max-width: 400px;
+  text-wrap: balance;
+}
+
+.v2-truck-lineup__title {
+  font-family: var(--ff-headline-medium);
+  font-size: 45px;
+  letter-spacing: -0.9px;
+  line-height: 117%;
+  margin: 0;
+}
+
+.v2-truck-lineup__buttons-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+  justify-content: center;
+  margin-top: 32px;
+}
+
+.v2-truck-lineup__buttons-container a.button,
+.v2-truck-lineup__buttons-container .button-container {
+  margin: 0;
+}
+
+/* Navigation */
+.v2-truck-lineup__navigation::-webkit-scrollbar {
+  display: none;
+}
+
+ul.v2-truck-lineup__navigation {
+  display: flex;
+  flex-flow: row nowrap;
+  list-style: none;
+  margin: 32px 0;
+  order: 0;
+  overflow: auto;
+  padding: 2px 32px;
+  position: relative;
+}
+
+.v2-truck-lineup__navigation-line {
+  background: var(--c-primary-black);
+  bottom: 3px;
+  height: 3px;
+  left: 0;
+  margin: 0;
+  position: absolute;
+  transition: all var(--duration-small) var(--easing-standard);
+  width: 0;
+}
+
+.v2-truck-lineup__navigation::before,
+.v2-truck-lineup__navigation::after {
+  content: '';
+  margin: auto;
+}
+
+.v2-truck-lineup__navigation-item {
+  border-bottom: 1px solid var(--c-primary-black);
+}
+
+.v2-truck-lineup__navigation-item.active button,
+.v2-truck-lineup__navigation-item button:hover {
+  --color-icon: var(--c-primary-black);
+
+  color: var(--c-primary-black);
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.v2-truck-lineup__navigation-item button {
```

Comment:
Instead of changing this button, I updated in styles, because by default button is using font-family Helvetica Roman, so the weight there was already wrong.